Release new version to crates.io

### Summary

Current version in crates.io have a link error while using the recent rust nightly.

https://github.com/rustwasm/wee_alloc/pull/56 fix this problem

### Steps to Reproduce

lib.rs:
```rust
use wasm_bindgen::prelude::*;
use web_sys::console;
#[wasm_bindgen]
pub fn init() {
    console::log_1(&format!("it run").into());
}
#[global_allocator]
static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
```
Cargo.toml
```
[dependencies]
wasm-bindgen = "0.2"
wee_alloc = "0.4.2"

[dependencies.web-sys]
version = "0.3"
features = ["console"]
```
### Actual Results

```
ERROR in ./pkg/wasm_test_bg.wasm
Module not found: Error: Can't resolve 'env' in '/mnt/g/project/wasm_test/pkg'
 @ ./pkg/wasm_test_bg.wasm
 @ ./pkg/wasm_test.js
 @ ./index.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 ./index.js
```
in wasm file
```
  (import "env" "llvm.wasm.grow.memory.i32" (func $llvm.wasm.grow.memory.i32 (type $t2)))
```
### Expected Results

Release a new version to crates.io

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,7 @@
 
 Released 2019/02/18.
 
+* Add support for building on stable Rust on Windows and Unix.
 * `wasm32` intrinsics are now invoked using `core::arch` rather than LLVM.
 * Use `SRWLOCK` for windows implementation.
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,10 @@
+### 0.4.3
+
+Released 2019/02/18.
+
+* `wasm32` intrinsics are now invoked using `core::arch` rather than LLVM.
+* Use `SRWLOCK` for windows implementation.
+
 ### 0.4.2
 
 Released 2018/07/16.

--- a/wee_alloc/Cargo.toml
+++ b/wee_alloc/Cargo.toml
@@ -11,7 +11,7 @@ license = "MPL-2.0"
 name = "wee_alloc"
 readme = "../README.md"
 repository = "https://github.com/rustwasm/wee_alloc"
-version = "0.4.2"
+version = "0.4.3"
 
 [badges]
 travis-ci = { repository = "rustwasm/wee_alloc" }

